src/Common/AsynchronousMetrics.cpp

```diff
@@ -929,7 +938,7 @@ void AsynchronousMetrics::processWarningForMemoryOverload(const AsynchronousMetr
         memory_resident = cgroup_memory_used->value;
         memory_total = cgroup_memory_total->value;
     }
-    else if (const auto * os_memory_used = getAsynchronousMetricValue(new_values, "OSMemoryResident"),
```

Comment:
Looks like this was changed accidentally in one of the commits in https://github.com/ClickHouse/ClickHouse/pull/86838.